src/bioregistry/data/bioregistry.json

```diff
@@ -53913,7 +53916,7 @@
       "name": "Charles Tapley Hoyt",
       "orcid": "0000-0003-4423-4370"
     },
-    "description": "ICD11 codes, which typically appear in health records and can be used for post-coordination",
+    "description": "An ICD-11 code is a standardized alphanumeric identifier used to classify and document diseases, conditions, and health-related issues for global healthcare and statistical purposes.",
```

Comment:
can you please give some more context?

Who uses ICD11 codes, and where? Why do they exist as a counterpart to ICD11 identifiers, which have URIs and everything, but these don't?